cmd/container-pipe/main.go

```diff
@@ -0,0 +1,79 @@
+package main
```

Comment:
if we decided that we didn't want to support the Update path, would this tool be something that we'd need to agree to maintain?

Reply:
The tool is mostly an example of it working. The proper spot to put it if we want it in a tool I think is in "shimdiag share" which already has a precedent for this ad-hoc call but isnt supported nicely via containerd. I think we should just remove the tool now that we know it will work via containerd update api. Only reason for tool I guess is non-containerd based approach, but I dont see why thats important anymore :)